game/src/contrib/components/InventoryComponent.java

```diff
@@ -11,47 +11,53 @@
 import java.util.logging.Logger;
 
 /**
- * This Component marks an {@link Entity} as having an inventory
+ * Allows the entity to collect items in an inventory.
  *
- * <p>It keeps track of the currently carried items in the list {@link #inventory} and the size of
- * the inventory in {@link #maxSize}. Carried items can be retrieved by using {@link #items()
- * getItems}. List elements can be added via {@link #addItem(ItemData) addItem} and removed via
- * {@link #removeItem(ItemData) removeItem}.
+ * <p>The component stores a collection of {@link ItemData} that are currently stored in the
+ * associated entity.
```

Comment:
Wer stored denn nun? Die Component oder die Entität (oder beide)? Das letzte "stored" liest sich für mich wie "associated", aber das kommt dann nochmal in Bezug auf die Entität-Component-Beziehung.

Reply:
konkretisiert